### ISSUE TITLE
Create a Logo for Jupyter RTC

### ISSUE THREAD

**Issue by echarles** [jupyterlab/rtc#51]
There are plenty of ways to decline the Jupyter logo to highligh the realtime collaboration features we are building here.

The source SVG can be found in the [design repo](https://github.com/jupyter/design/tree/master/logos).

e.g. ![alt text](https://raw.githubusercontent.com/jupyter/design/master/logos/Square%20Logo/squarelogo-greytext-orangebody-greymoons/squarelogo-greytext-orangebody-greymoons.svg)

Just comment on this issue with your SVG source file. I guess the comment with the more 👍 will be the official one.

**Comment by echarles** [comment]
GitHub issue does not support SVG attachment... We accept PNG assuming you will ship the SVG format if you are selected... Good luck!

**Comment by echarles** [comment]
Proposal created by @eleonorecharles
![jupyter-rtc_1](https://user-images.githubusercontent.com/226720/114676864-08ee4980-9d0a-11eb-8be1-ae909154dc12.png)

**Comment by echarles** [comment]
Proposal created by @eleonorecharles
![jupyter-rtc_2](https://user-images.githubusercontent.com/226720/114676899-0f7cc100-9d0a-11eb-9e12-a44df163fa60.png)

**Comment by echarles** [comment]
Proposal created by @eleonorecharles
![jupyter-rtc_3](https://user-images.githubusercontent.com/226720/114676924-14417500-9d0a-11eb-93b5-d86b115a28a5.png)

**Comment by echarles** [comment]
Proposal created by @eleonorecharles
![jupyter-rtc_4](https://user-images.githubusercontent.com/226720/114676946-199ebf80-9d0a-11eb-89fd-0c367d24fcd1.png)

**Comment by jtpio** [comment]
We might want to double check these are conform to the guidelines.

In particular the "Logo Misuses" section in this document: https://github.com/jupyter/design/blob/master/brandguide/brand_guide.pdf

**Comment by echarles** [comment]
Thx for the pointer @jtpio. I guess all contributors are not designers and the selected one should go through a more formal process to be officially announced and published. A review in terms of accessibility will also be needed.

**Comment by Carreau** [comment]
> I guess all contributors are not designers and the selected one should go through a more formal process to be officially announced and published.

This may not be sufficient, or possible. Creating official derivative of the logo can void the Jupyter trademark.
In general that is why many project have 2 logos, only one of which is trademarked. For example the android logo and the android robot. The trademark android logo has just the font with the word "Android" and cannot be modified even for official use. The robot though is not;  and can have derivative work.

I'm not part of the trademark committee anymore, I would suggest contacting them, but I would strongly advise against modifying the jupyter logo for a RTC version. 

Note (I'll also edit your comments to number the proposal so that we can refer to it in test.)

**Comment by echarles** [comment]
@Carreau I am trying to find a contact for the trademark on eg https://jupyter.org/governance/trademarks.html but can not find any. Do you have any pointer

**Comment by Carreau** [comment]
to me proposal 1 is the more balanced, but it screams Ubuntu "Circle of friends" (official name) . The other ones as  well but to a lesser extent. I think that simply the dot in the middle of the arc is in my opinion  close enough to the shape of one friends in the "circle of friends" that it could be deemed too confusing to be part of the logo.

**Comment by Carreau** [comment]
Also Who the F**** altered the original SVG logo , the circles are not circles anymore in the SVG you pointed to.

See comparison of current svg in the design repo vs previous one:
<img width="293" alt="Screen Shot 2021-04-14 at 08 59 45" src="https://user-images.githubusercontent.com/335567/114741650-d3446300-9cff-11eb-80fc-9149479cf4e1.png">

**Comment by echarles** [comment]
> Also Who the F**** altered the original SVG logo , the circles are not circles anymore in the SVG you pointed to.

@Carreau  I remember github was allowing to upload SVG like in the initial comment (17 Jul 2020), but today it does not allow it anymore. Maybe the distortion you have found the reason why github has disabled that feature...

**Comment by Carreau** [comment]
> I remember github was allowing to upload SVG like in the initial comment (17 Jul 2020), but today it does not allow it anymore. Maybe the distortion you have found is the reason why github has disabled that feature...

Maybe, I think the original comment is just a link to the current file on master, and master was changed since then. It's not relevant to the current conversation too much it's just weird.

**Comment by willingc** [comment]
Hmm... @echarles Thanks for doing the work to create options. I'm not sure about adding a new logo for RTC. Isn't RTC going to be a standard part of JupyterLab?

**Comment by echarles** [comment]
> Hmm... @echarles Thanks for doing the work to create options. I'm not sure about adding a new logo for RTC. Isn't RTC going to be a standard part of JupyterLab?

... of JupyterLab but also of Jupyter Server, JupyterHub... or as a component to be consumed by any third party application. For now we are working on a nbmodel that resides indeed under the jupyterlab packages but this can evolve if it makes sense.

So yes, this may be out of scope for now... sorry if I bring confusion with this... I thought it was also a way to do something else and maybe interest other people...

**Comment by willingc** [comment]
Thanks @echarles for providing a bit more info on the scope of the usage. No confusion brought at all. I'm all for fun and making things interesting.

**Comment by ellisonbg** [comment]
The contact information for the trademark committee is: jupyter-trademarks@googlegroups.com

The trademark committee has considered a number of requests from individuals and organizations asking to modify the Jupyter logo. So far, the general trend of the trademark committee has been to ask people to "use but don't modify" the logo, which is also the recommendation in the branding guideline above.

**Comment by echarles** [comment]
Thx @ellisonbg This is the mail I have just sent to jupyter-trademarks@googlegroups.com

Dear All,

I'd like to submit to your review the idea around a specific logo for Jupyter Realtime Collaboration.

https://github.com/jupyterlab/rtc/issues/51

Can you please comment on that issue what we should do for the next steps: Or close, or further process with the idea  or ...

Kr, Eric

**Comment by echarles** [comment]
The idea of a dedicated logo has come as eg. jupyterhub, jupyterlab with their own logo have been helpful to me (as explained on https://github.com/jupyter/design/issues/65). 

**@ all /  Please wait on the trademark committee answer before working or submitting on new proposal. We may close this and forget the idea.**
